map_model/src/objects/road.rs

```diff
@@ -269,6 +269,12 @@ impl Road {
             .min_by_key(|(idx, _)| (our_idx - (*idx as isize)).abs())
             .map(|(_, l)| l)
     }
+    /// This function returns the lane closest to the center from the left hand side of the street
+    pub fn get_lane_underneath_label(&self) -> LaneType {
```

Comment:
The `map_model` layer of code shouldn't really "know" about rendering. So how about we named this `get_middle_lane` or `get_center_lane` instead? Defining what the middle/center means is a bit vague, but the docstring is great